src/helix/client/client_ext.rs

```diff
@@ -1220,6 +1220,22 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
             .try_flatten_unordered(None)
     }
 
+    /// Update the user's description
+    ///
+    /// The user ID in the OAuth token identifies the user whose information you want to update.
+    pub async fn update_user<'b, T>(
```

Comment:
maybe `update_user_description` ? Since twitch could add new fields to update (like panels, pls)

Reply:
Yea I had that idea as well. I'll add a new ignited suffix to the documentation collector.